app/controllers/hub/efile_submissions_controller.rb

```diff
@@ -8,6 +8,7 @@ class EfileSubmissionsController < ApplicationController
 
     def index
       @efile_submissions = EfileSubmission.most_recent_by_tax_return.page(params[:page])
+      @efile_submissions = @efile_submissions.in_state(params[:status]) if params[:status].present?
```

Comment:
very quick implementation to allow for filtering by status on efile submission page, otherwise the page becomes pretty useless. No UI -- that will come later.